android/KMEA/build.sh

```diff
@@ -98,7 +98,7 @@ fi
 
 if builder_start_action test:engine; then
 
-  if builder_has_option --ci; then
+  if builder_is_ci_test_build; then
```

Comment:
`builder_is_ci_test_build` instead of `builder_is_ci_build` for reporting test results to CI

Reply:
I think we should still be reporting unit test results to CI, shouldn't we? Even for release builds? If they fail, we want to see why.

Reply:
Right. Isn't `builder_is_ci_test_build` for test builds on CI?

Reply:
A release build isn't a test build, but if there are unit tests that fail we still want to know about them (because they still run on release builds for sanity).